packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -490,6 +497,7 @@ class ScrollPosition extends ViewportOffset {
     _activity = newActivity;
     if (oldIgnorePointer != shouldIgnorePointer)
       state.setIgnorePointer(shouldIgnorePointer);
+    isScrollingNotifier.value = _activity?.isScrolling ?? false;
```

Comment:
no need for all the elvis, activity can't be null by here (because this function early-returns on null)